### PR TITLE
New shopping page

### PR DESCRIPTION
**Pull request by aquaopaque**

### CODE REVIEW

**Comment by gregwym** [comment]
来个截图

File: static/styles/home.css [hunk 1]
```diff
@@ -69,3 +72,157 @@
...
+    margin: 1.5em;
+}
+.sec-common {
+    background-size: contain;
+    position: relative;
+    width: 100%;
+}
+.sec-common> img {
+    max-width: 100%;
+    height: auto;
```
Comment (gregwym) [inline static/styles/home.css:113]: Add `min-width: 800px` here.

File: static/styles/home.css [hunk 2]
```diff
@@ -1,5 +1,6 @@
 .container {
-  width: 800px;
+    min-width: 580px;
```
Comment (gregwym) [inline static/styles/home.css]: Get rid of this `min-width` for `.container`. Just let it flow.

File: static/styles/home.css [hunk 3]
```diff
@@ -69,3 +72,157 @@
...
+    font-size: 1vw;
+    top: 2%;
+}
+#sec-slogan1 {
+    _background: url('/static/imgs/intro/slogan1.jpg') no-repeat;
+}
+#sec-slogan1 div {
+    width: 23em;
+    right: 6em;
+    padding-top: 3em;
```
Comment (gregwym) [inline static/styles/home.css]: Try `padding-top:10%`. I think this works better for both small and large screen. same for `#sec-slogan2 div` and `#sec-slogan3 div`.

File: static/styles/home.css [hunk 4]
```diff
@@ -69,3 +72,157 @@
...
+    _background: url('/static/imgs/intro/slogan4.jpg') no-repeat;
+}
+#sec-slogan4 div.outer {
+    position: absolute;
+    left: 50%;
+}
+#sec-slogan4 div.inner {
+    color: white;
+    position: relative;
+    padding-top: 4%;
```
Comment (gregwym) [inline static/styles/home.css]: Remove this `padding-top` and add `padding-top: 10%` to `#sec-slogan4 div.out`.

**Comment by gregwym** [comment]
qq20140327-2
qq20140327-1
Should be good after all the changes above. Don't blame me if it won't look the same on phone or IE.

**Comment by aquaopaque** [comment]
Apply your changes directly to the code. No need to inform me if it has nothing to do with layout or functionality. Thanks.

**Comment by gregwym** [comment]
Sure I can fix it myself, but I have no access to your branch. Just trying to put everything in one place.

**Comment by aquaopaque** [comment]
Create a patch file and send to me

**Pull request merged**